db/sqlc/migrations/000001_accounts.up.sql

```diff
@@ -0,0 +1,71 @@
+CREATE TABLE IF NOT EXISTS accounts (
+    -- The auto incrementing primary key.
+    id INTEGER PRIMARY KEY,
+
+    -- The ID that was used to identify the account in the legacy KVDB store.
+    -- In order to avoid breaking the API, we keep this field here so that
+    -- we can still look up accounts by this ID for the time being.
+    alias BLOB NOT NULL UNIQUE,
```

Comment:
do we need to use blob here or can this be a more efficient type, maybe a bigint?

Reply:
good idea!

Reply:
updated